recipes/biobambam/build.sh

```diff
@@ -1,12 +1,8 @@
 #!/bin/bash
 set -eu
+./configure --prefix=${PREFIX}
 
-mkdir -p $PREFIX/bin
-export CPP_INCLUDE_PATH=${PREFIX}/include
-export CPLUS_INCLUDE_PATH=${PREFIX}/include
-export CXX_INCLUDE_PATH=${PREFIX}/include
-export LD_LIBRARY_PATH=${PREFIX}/lib
-export LIBRARY_PATH=${PREFIX}/lib
+#debug
+cat config.log
```

Comment:
Don't think we need to keep these lines?

Reply:
removed